hack/make/build-rpm

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+
+# subshell so that we can export PATH without breaking other things
+(
+	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
+
+	# TODO consider using frozen images for the dockercore/builder-rpm tags
+		
+	rpmName=docker-engine
+	rpmVersion="${VERSION%%-*}"
+	rpmRelease=1
+
+	# rpmRelease versioning is as follows
+	# Docker 1.7.0:  version=1.7.0, release=1
+	# Docker 1.7.0-rc1: version=1.7.0, release=0.1.rc1
+	# Docker 1.7.0-dev nightly: version=1.7.0, release=0.0.YYYYMMDD.HHMMSS.gitHASH
+	
+	# if we have a "-rc*" suffix, set appropriate release
+	if [[ "$VERSION" == *-rc* ]]; then
+		rcVersion=${VERSION#*=rc}
```

Comment:
What is `#*=rc` supposed to be doing here?  Stripping off the `1.7.0-rc`?  I think you probably meant `-` instead of `=` there. :+1: